pkg/adaptation/default-validator.go

```diff
@@ -0,0 +1,173 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package adaptation
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/containerd/nri/pkg/log"
+	yaml "gopkg.in/yaml.v3"
+)
+
+type DefaultValidatorConfig struct {
+	Enable         bool `toml:"enable"`           // enable default validator
```

Comment:
Why might we want to configure this in the plugin (vs. specifying in container runtime config what default NRI plugins we want disabled?)

Reply:
It is going to come from the runtime's configuration. It's just that, since we have two NRI-enabled runtimes, we define here a single common data structure for both to use and pass the validation related configuration bits to NRI. Here is my [containerd devel/test branch](https://github.com/klihub/containerd/commits/devel/nri-pluggable-validation) with pluggable validation for the details.